.github/workflows/ci.yml

```diff
@@ -727,6 +727,10 @@ jobs:
         if: matrix.os == 'windows'
         uses: microsoft/setup-msbuild@v2
 
+      - name: Install fpm
+        if: runner.os == 'Linux'
```

Comment:
minor nit: maybe use matrix.os; it doesn't really matter here but in a cross compilation environment we could be building on a Linux runner but targeting another OS. The matrix.os holds the OS we are targetting.